src/Domain/SkillSetDomain.php

```diff
@@ -23,7 +23,7 @@ public function __construct(array $availableSkillSets = [], int $importantSkills
      */
     public function getSkillSet(): array
     {
-        return $this->availableSkillSets;
+        return array_map(fn (array $skill) => $skill['label'], $this->availableSkillSets);
```

Comment:
you can use `array_column`

Reply:
actually i can't: we must preserve keys, otherwise i should use `array_column` in combination with `array_combine` and `array_keys`, but this seems more complicated than a simple array_map